src/ShopBundle/objects/WebModules/MTShopBasketCore/MTShopBasketCoreEndpoint.class.php

```diff
@@ -941,10 +939,8 @@ public function UpdateBasketItem($aRequestData = null, $bAddAmountToExistingAmou
             } else {
                 $bItemWasUpdated = $oBasket->AddItem($oArticle);
             }
-            if (null !== $oExistingItem) {
```

Comment:
Isn't this opening brace now missing? Or is it just the diff display?

Reply:
we removed 2 opening and one closing brace - and replaced it with one opening brace. So no, resulting number of braces is ok (moved from 2 opening and 2 closing to one opening and one closing).